src/modules/ekf2/EKF/common.h

```diff
@@ -281,7 +289,7 @@ struct parameters {
 	float initial_tilt_err{0.1f};           ///< 1-sigma tilt error after initial alignment using gravity vector (rad)
 
 #if defined(CONFIG_EKF2_BAROMETER)
-	int32_t baro_ctrl{1};
+	int32_t baro_ctrl {1};
```

Comment:
why do we need a space here and not on the other variables?

Reply:
I don't think we do, but at the moment it's a compromise to let astyle do the rest automatically.

We could try and fix this particular little annoyance, but I don't think that should block the change for now. 
https://astyle.sourceforge.net/astyle.html